### DISCUSSION TITLE
💋 【共学营打卡】【第6次 Milestone】部署 Sui 上的 Sandwich 合约并成功调用

### DISCUSSION THREAD

**Discussion by leeduckgo** [NonceGeek/Move-Camp#278] [Projects-Others]
> https://mp.weixin.qq.com/s/9PGG9G8oaPUeZYkZNjfbxg

**Comment by MagicGordon** [comment]
![6-1](https://user-images.githubusercontent.com/19465870/217523871-4198ef9c-8399-4ca8-adc0-c3669d37e4a5.png)
![6-2](https://user-images.githubusercontent.com/19465870/217523901-b35195e1-343f-4534-be1e-c5249e2f2d68.png)

**Comment by xiegudong45** [comment]
部署：https://explorer.sui.io/object/0x411241619eb8cabb4b5ede740f3cb4174c743194?network=devnet
![image](https://user-images.githubusercontent.com/30063330/218299882-adfbd848-0863-4057-ad90-fdf051a259ae.png)
调用合约：https://explorer.sui.io/transaction/3F56dWDaz94ht1RG6XTHdMmr4mzuxLDUngFB7CEDwH5X?network=devnet
![image](https://user-images.githubusercontent.com/30063330/218300214-6003f8c1-6479-4517-8707-7793ef74dc8a.png)

**Comment by baidang201** [comment]
![image](https://user-images.githubusercontent.com/3405789/218302085-015636f5-0169-4d1d-a46d-984598c0bb24.png)
![image](https://user-images.githubusercontent.com/3405789/218302095-e0b34dcd-6a92-4890-a86d-2e1e45cc4819.png)
